app/scripts/main.tsx

```diff
@@ -56,6 +58,8 @@ const composingComponents = [
   LayoutRootContextProvider
 ];
 
+const useNewExploration = !!process.env.FEATURE_NEW_EXPLORATION;
```

Comment:
One more suggestion to use an explicit value for this environment variable, but other than that I think this should be good to go!

Reply:
Added a simple function to be used in cases like this `checkEnvFlag(process.env.VAR_NAME)`